src/components/CustomFieldsPlanAlert.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useCurrentUser } from 'common/hooks/useCurrentUser';
+import { useTranslation } from 'react-i18next';
+import { Alert } from './Alert';
+import { Link } from './forms';
+import CommonProps from '../common/interfaces/common-props.interface';
+import { proPlan } from 'common/guards/guards/pro-plan';
+import { enterprisePlan } from 'common/guards/guards/enterprise-plan';
+import { isHosted } from 'common/helpers';
+import { MdInfoOutline } from 'react-icons/md';
+
+export function CustomFieldsPlanAlert(props: CommonProps) {
+  const [t] = useTranslation();
+
+  const user = useCurrentUser();
+
+  const showAlert = !proPlan() && !enterprisePlan() && isHosted();
+
+  return (
+    <>
+      {showAlert && (
+        <div className={`${props.className}`}>
```

Comment:
No need to encapsulate the variable here.

```ts
<div className={props.className}>
```

should work just fine.

Reply:
Sure, just changed!